SU-HSDO/suhumsci
# SHS-5811: Add logout button to sites

# READY FOR REVIEW

## Summary
- Adds a temporary `composer.json` change for adding a forked version of the `stanford_samlauth` module that adds logout button functionality. (This change needs to be reverted before merge!)
- Adds classes to the logout button to match login button

## Need Review By (Date)
2024-09-07

## Urgency
low

## Steps to Test
1. Login to the site as an admin
2. Visit the Block layout page Block page: `/admin/structure/block`
3. Click "Place block" in the footer region
4. Choose the "SAML SUNetID Logout Block"
5. Do not display the title, then click "Save Block"

![Screenshot 2024-10-15 at 4 06 34 PM](https://github.com/user-attachments/assets/27e27ff8-8ec2-40d4-8d07-05a0d5111e7e)

6. Verify on the front-end, while logged in, that there is now a logout button

![Screenshot 2024-10-15 at 4 05 15 PM](https://github.com/user-attachments/assets/41b38d32-d024-408f-8e81-15cfc46ef51d)

7. Click o the logout button and verify that it correctly logs you out of the site

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208463370275106
